### ISSUE TITLE
Source of `extension-page-icon.png` missing ?

### ISSUE THREAD

**Issue by zapashcanon** [hslbck/gnome-shell-extension-radio#47]
Hi,

I just noticed this using check-all-the-things:

```sh
# Check with upstream where the Inkscape SVG source files are.
$ find -type f \( -iname '*.png' -o -iname '*.gif' -o -iname '*.jpg' -o -iname '*.jpeg' \) -exec grep -iF inkscape {} +
Fichier binaire ./radio@hslbck.gmail.com/icons/extension-page-icon.png correspondant
```

@hslbck are you the author of it ? If yes, do you still have the source ? If not, do you remember where did you get it ? :)

**Comment by hslbck** [comment]
Hi,

the source is available, see https://github.com/hslbck/gnome-shell-extension-radio/blob/master/radio%40hslbck.gmail.com/icons/extension-page-icon.png

**Comment by zapashcanon** [comment]
Yes, but it's a `.png`, by source I mean the corresponding `.svg` file.

**Comment by hslbck** [comment]
ah ok, then the real source is https://github.com/hslbck/gnome-shell-extension-radio/blob/master/radio%40hslbck.gmail.com/icons/gser-icon-stopped-symbolic.svg. The png file is just an export of this svg with the color black. It has no separate svg source file

**Comment by zapashcanon** [comment]
```sh
cat radio@hslbck.gmail.com/* | grep extension-page-icon
```

Gives me no output, looks like it's not used, what's this icon for ?

**Comment by hslbck** [comment]
I just packed the all sources together. It is used on https://extensions.gnome.org/extension/836/internet-radio/

**Comment by zapashcanon** [comment]
I see, thanks. I'll probably add a .svg for the file then, so that, check-all-the-things will stop complaining about it. :)

**Comment by zapashcanon** [comment]
@hslbck, I just tried to open `gser-icon-playing-symbolic.svg` with inkscape to change the color to black and save it as a new source file, but, got an error saying something like `Unable to load file /[...]/gser-icon-playing-symbolic.svg`.

I never used inkscape before so, am I missing something here ?

**Comment by hslbck** [comment]
currently I'm getting the same error. Strange, because I created those files with inkscape

**Comment by zapashcanon** [comment]
Ok, I though it was because I changed links inside the .svg files to switch to https, so I reverted the changes, but got the same error.

**Comment by zapashcanon** [comment]
Arf no, it was because of this... reverting this change fixed it, I did it wrong the first time.

**Comment by hslbck** [comment]
I had to update the images anyway :D There were some svg changes by inkscape itself
